api/api.gradle.kts

```diff
@@ -11,7 +11,7 @@ dependencies {
     api(group = "org.fxmisc.easybind", name = "easybind", version = "1.0.3")
     api(group = "org.controlsfx", name = "controlsfx", version = "8.40.11")
     api(group = "de.codecentric.centerdevice", name = "javafxsvg", version = "1.2.1")
-    api(group = "edu.wpi.first.ntcore", name = "ntcore-java", version = "3.1.7-20170808143930-12-gccfeab5")
+    api(group = "edu.wpi.first.ntcore", name = "ntcore-java", version = "4.+")
     implementation(group = "edu.wpi.first.wpiutil", name = "wpiutil-java", version = "2.0.0-20170808143537-16-gf0cc5d9")
-    implementation(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "3.1.7-20170808143930-12-gccfeab5", classifier = "all")
+    implementation(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "4.+", classifier = "all")
```

Comment:
I'm assuming there was a meeting on this and this was a decision made all around?

Reply:
Yes